packages/open-next/src/build/bundleNextServer.ts

```diff
@@ -42,7 +42,11 @@ const externals = [
   "next/dist/compiled/next-server",
 ];
 
-export async function bundleNextServer(outputDir: string, appPath: string) {
+export async function bundleNextServer(
```

Comment:
FYI this function should not be necessary in the future. Next should provide us directly with an already bundled `NextServer` in the future